ang/api4Explorer/Explorer.html

```diff
@@ -35,6 +35,14 @@ <h1 crm-page-title>
               <label for="api4-param-selectRowCount">SelectRowCount</label>
             </div>
           </div>
+          <div class="api4-input form-inline" ng-mouseenter="help(name, param)" ng-mouseleave="help()" ng-repeat="(name, param) in availableParams" ng-if="!isSpecial(name) && param.type[0] === 'bool' && param.default === null">
+            <label for="api4-param-{{ name }}">{{ name }}<span class="crm-marker" ng-if="param.required"> *</span></label>
+            <select class="form-control" id="api4-param-{{ name }}" ng-model="params[name]">
+              <option ng-value="null"></option>
+              <option ng-value="true">True</option>
+              <option ng-value="false">False</option>
```

Comment:
@totten I'm stuck here - according to the [anugular docs](https://code.angularjs.org/1.5.11/docs/api/ng/directive/ngValue), this should work, but it doesn't. Can you think of what might be wrong?